src/main/java/org/team1515/botmitzvah/Controls.java

```diff
@@ -0,0 +1,68 @@
+package org.team1515.botmitzvah;
+
+import java.util.function.BooleanSupplier;
+
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
+public class Controls {
+    // change botton maps later
+    public static final Trigger GRAB = new Trigger(RobotContainer.mainController::getXButton);
+    public static final Trigger ALIGN = new Trigger(RobotContainer.mainController::getStartButton); // will limelight
+                                                                                                    // and apriltag
+                                                                                                    // overlap?
+    public static final Trigger RESET_GYRO = new Trigger(RobotContainer.mainController::getBackButton);
+    public static final BooleanSupplier DRIVE_ROBOT_ORIENTED = () -> RobotContainer.mainController.getLeftBumper();
+
+    // Climber Buttons
+    // public static final Button EXPAND_VERTICAL = new
+    // Button(OI.mainStick::getRightBumper);
+    // public static final Button RETRACT_VERTICAL = new
+    // Button(Controls::getRightTrigger);
+    // public static final Button EXPAND_DIAGONAL = new
+    // Button(OI.mainStick::getYButton);
+    // public static final Button RETRACT_DIAGONAL = new
+    // Button(OI.mainStick::getXButton);
+
+    // public static final Button MANUAL_CLIMBER_L = new
+    // Button(OI.mainStick::getBButton);
+    // public static final Button MANUAL_CLIMBER_R = new
+    // Button(OI.mainStick::getAButton);
+
+    // public static final Button LEFT_DPAD = new
+    // Button(DPadButton.LEFT::getDPadButton);
+    // public static final Button RIGHT_DPAD = new
+    // Button(DPadButton.RIGHT::getDPadButton);
+    // public static final Button UP_DPAD = new
+    // Button(DPadButton.UP::getDPadButton);
+    // public static final Button DOWN_DPAD = new
+    // Button(DPadButton.DOWN::getDPadButton);
```

Comment:
if you want to keep this i'd rename it or remove it for the time being, given that the buttons themselves are already defined